### ISSUE TITLE
Thin Version

### ISSUE THREAD

**Issue by dnlbrg76** [d8-bbc11/monolisa#6]
It seems to be the thin font

**Comment by d8-bbc11** [comment]
i don't wanna be that guy but It Works On My Machine™
try using the 'MonoLisa' weight, see if it works

**Comment by deFrogdev** [comment]
yeah bro its only contains (bold - italic - light) no regular font
![image](https://github.com/d8-bbc11/monolisa/assets/147989026/93e786b9-03ec-4d1a-878f-6b9bc6925036)

**Comment by d8-bbc11** [comment]
i cant reproduce this issue, neither on windows or on linux but you could try opening the font file in [FontForge](https://fontforge.org/) and exporting it
tell if it works

**Comment by deFrogdev** [comment]
sorry dont know how that program works!

**Comment by dnlbrg76** [comment]
> i cant reproduce this issue, neither on windows or on linux but you could try opening the font file in [FontForge](https://fontforge.org/) and exporting it tell if it works

This is weird... Downloading with Chrome for Windows -> thin version
Downloading same file with Chrome on Android -> normal version

**Comment by grousk** [comment]
On linux try running `fc-list MonoLisa` it should show ALL the weights.

**Comment by d8-bbc11** [comment]
> sorry dont know how that program works!

i LITERALLY gave you instructions

1. get fontforge
2. open the file
3. export it (to a TTF file)
4. done

![](https://media.tenor.com/IsneokwQDf8AAAAM/george-floyd-truly-lawst.gif)

> This is weird... Downloading with Chrome for Windows -> thin version Downloading same file with Chrome on Android -> normal version

androids based on linux, that's prolly the reason it works there

**Comment by HuyTranHoang** [comment]
JFYJ. I have the same problem when use with IntelliJ IDEA. But in my case it only show with "bold". And I can't see any options in Typography Setting. ( My window fonts folder has every font weight )

I try to install fontforge, export it to a TTF, reinstall  and got it working with regular weight.

Thank you.

![image](https://github.com/d8-bbc11/monolisa/assets/20708669/4aecc109-5d4e-4df0-bbdc-ec17d199da96)

**Comment by vetoh65255** [comment]
can anyone add regular font because there is 2 light and light italic font but no regular or regular italic font available.
